clipboard.mod/doc/tclipboard_hasownership.bmx

```diff
@@ -0,0 +1,18 @@
+SuperStrict
+
+Framework Brl.Clipboard
+Import Brl.StandardIO
+
+' create a clipboard manager to access the system wide clipboard
+Local clipboard:TClipboard = new TClipboard.Create()
+
+' clipboard mode can be:
+' LCB_CLIPBOARD = The primary (global) clipboard [default mode]
+' LCB_SELECTION = The (global) mouse selection clipboard.

```

Comment:
Hmm. I think I broke your inline docs with the latest commit...
There is now `LCB_CLIPBOARD`, `LCB_PRIMARY` and `LCB_SECONDARY`.